opm/simulators/timestepping/SimulatorTimer.cpp

```diff
@@ -59,22 +61,24 @@ namespace Opm
     void SimulatorTimer::init(const ParameterGroup& param)
     {
         const int num_psteps = param.getDefault("num_psteps", 1);
+        const int end_step = param.getDefault("end_step", 50000);
```

Comment:
This entire constructor is obsolete and can be removed. Did not notice it until this PR but it has been unused for a long time.